app-system/sources.list

```diff
@@ -3,9 +3,9 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 # A collection of fully FLOSS app definitions and FLOSS apps for Citadel.
-https://github.com/runcitadel/apps v3-beta
+https://github.com/runcitadel/apps v3-stable
```

Comment:
Did you mean to make this change? It seems curious, lol.. just making sure!

Reply:
No, this was a mistake, it should be `v4-dev` (and later `v4-beta` and `v4-stable`)